### PR TITLE
Add gpcal

### PR DESCRIPTION
**Pull request by jhparkastro**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/gpcal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+requirements:
+  host:
+    - astropy
+    - matplotlib
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy
```
Comment (marcelotrevisani) [inline recipes/gpcal/meta.yaml:24]:
```suggestion
  host:
    - pip
    - python
```
I believe here you just need ``python`` and ``pip``

File: recipes/gpcal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  run:
+    - astropy
+    - matplotlib
```
Comment (marcelotrevisani) [inline recipes/gpcal/meta.yaml:27]:
```suggestion
    - matplotlib-base
```

File: recipes/gpcal/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ca87d0418e51cb9b7334b2d48a6102c2730eeb687e4a6a5f85585e7ea98cda07
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/gpcal/meta.yaml:14]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

**Comment by jhparkastro** [comment]
Hi, thank you so much. I have updated the file as you suggested.

File: recipes/gpcal/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    - gpcal
+
+about:
+  home: "https://github.com/jhparkastro/gpcal"
+  license: "GPL-2.0-or-later"
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: "A generalized instrumental polarization calibration pipeline for VLBI data"
+  doc_url: 
+  dev_url:
```
Comment (marcelotrevisani) [inline recipes/gpcal/meta.yaml:39]: Could you please remove these empty sections?
```suggestion
```

File: recipes/gpcal/meta.yaml (hunk 3 above)
Comment (jhparkastro) [reply recipes/gpcal/meta.yaml:14]: Done! Thanks!

File: recipes/gpcal/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    - scipy
+test:
+  imports:
+    - gpcal
+
+about:
+  home: "https://github.com/jhparkastro/gpcal"
+  license: "GPL-2.0-or-later"
+  license_family: GPL
+  license_file: LICENSE.txt
```
Comment (marcelotrevisani) [inline recipes/gpcal/meta.yaml:36]: It seems this project is not packaging the license, so you will need to add it manually

**Pull request closed without merging**